cip.go

```diff
@@ -196,7 +215,7 @@ func main() {
 	// If there are no images in the manifest, it may be a stub manifest file
 	// (such as for brand new registries that would be watched by the promoter
 	// for the very first time).
-	if len(*snapshotPtr) == 0 {
+	if doingPromotion && len(*manifestBasedSnapshotOf) == 0 {
```

Comment:
Is `doingPromotion` is actually doing anything here?  This looks like the only place you read the var, and it looks like it must be true when we get here anyway

Reply:
Hmm it's only true if we pass in `-manifest` or `-manifest-dir`.